Documentation/ceph-fs-subvolumegroup.md

```diff
@@ -35,7 +35,11 @@ If any setting is unspecified, a suitable default will be used automatically.
 
 ### CephFilesystemSubVolumeGroup metadata
 
-- `name`: The name that will be used for the Ceph Filesystem subvolume group.
+* `name`: The name that will be used for the Ceph Filesystem subvolume group.
+* `external`:
+  * `enable`: if `true` the cluster is external and we want to take advantage of the subvolumegroup
+    group feature along with the ceph-csi driver configuration for it.
```

Comment:
`subvolumegroup group feature along with the ceph-csi driver configuration for it` to `subvolumegroup feature along with the Ceph-CSI driver configuration for it`